stages/org.osbuild.luks2.format

```diff
@@ -150,6 +154,9 @@ def main(devices, options):
     if sector_size:
         command += ["--sector-size", str(sector_size)]
 
+    if integrity:
```

Comment:
Please excuse my python ignorance, but is `"none"` parsed to `None`? If not, does this give difference in behaviour between setting `"integrity": "none"` and leaving `integrity` unset?

Reply:
As in, the cryptsetup binary happily accepts "none" to mean the same as not specifying --integrity.

Reply:
Right, got it. I'd either drop "none", or make it the default value and then `if integrity != "none":`, just so we don't need to rely on `cryptseutp` doing the right thing.

Reply:
Lets drop it then, so we get the same cli args on old manifest files.